service/user.go

```diff
@@ -45,19 +45,15 @@ func (s *UserService) Delete(ctx context.Context, id uuid.UUID, authUserID uuid.
 		return err
 	}
 
-	if err := s.repository.Delete(ctx, id); err != nil {
```

Comment:
Yes, I just wanted to have it more consistent with the other domains. Because when deleting from `auth.users`, supabase provides endpoint for it and returns 404 if user does not exist. However, this is not the case when deleting from 'normal' tables, where I must first call a read function to check if the record exists so I can return a 404 error if it does not.